api/core/storage/slack/slack_api_client.py

```diff
@@ -15,63 +20,115 @@ def _client(self):
             headers={"Authorization": f"Bearer {self.bot_token}", "Content-Type": "application/json; charset=utf-8"},
         )
 
-    async def create_channel(self, name: str) -> str:
-        """Create a slack channel and return a channel id"""
+    def _check_response(
+        self,
+        parsed: dict[str, Any],
+        operation_name: str,
+    ) -> None:
+        """Check if Slack API response has ok=True, log and possibly raise error if not"""
+        if not parsed.get("ok", False):
+            error_msg = f"Slack client failed to {operation_name}"
+            error = parsed.get("error", "Unknown error")
+
+            _logger.error(error_msg, extra={"error_msg": error})
```

Comment:
no need to log the error since you are raison gbelow

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/152/commits/629b4ccb49d230acf8348a0ba2a2cf0f88627c10